prototype/src/main/webapp/chooseLocation.html

```diff
@@ -9,6 +9,12 @@
   <body onload="createMap();">
     <h1>Meeting Locations: </h1>
     <p>Click on the map to propose a new location.</p>
-    <div id="map"></div>
+    <div id="map"></div></br>
+    <button id="myButton" class="float-left submit-button" >Send Event</button>
```

Comment:
What are these classes for?

Reply:
It was from an example online but I don't think it is necessary, I will remove it